commitlint.config.js

```diff
@@ -19,9 +19,10 @@ module.exports = {
         'infrastructure',
         'infrastructure/build',
         'manager',
-        'manager/linux',
-        'manager/mac',
-        'manager/windows',
+        'manager/electron',
```

Comment:
This change seems a little pointless since there's only electron for the manager. The commitlint doesn't need to match the paths and we're not getting any more info with the extra `electron` bit.

Reply:
Yeah. Even then I might argue that `manager/www` can live alongside `manager/android`, etc.